contracts/chain-adapters/Arbitrum_Adapter.sol

```diff
@@ -161,17 +163,28 @@ contract Arbitrum_Adapter is AdapterInterface {
 
     ArbitrumL1ERC20GatewayLike public immutable l1ERC20GatewayRouter;
 
+    /**
+     * @notice Domain identifier used for Circle's CCTP bridge on Arbitrum.
+     * @dev This identifier is assigned by Circle and is not related to a chain ID.
+     * @dev Official domain list can be found here: https://developers.circle.com/stablecoins/docs/supported-domains
+     */
+    uint32 public constant circleDomainId = 3;
```

Comment:
nit: since this is only used to pass up through the constructor stack, I would suggest making it a private variable and name it ARBITRUM_CIRCLE_DOMAIN_ID (I think it's good to include the chain name so it's very clear that the constant is chain specific). It will be exposed via the recipientCircleDomainId in the parent class, so we don't really gain anything from exposing with a public function here.

Same for the other adapters.

Reply:
Updated here: https://github.com/across-protocol/contracts-v2/pull/391/commits/dd5e3f8719efe998c811d09831c56fcae71bac4a